core/src/components/alert/test/basic/index.html

```diff
@@ -284,7 +285,8 @@
           inputs: [
             {
               type: 'checkbox',
-              label: 'Checkbox 1',
+              label:
```

Comment:
Could we do this in a separate test or move the disabled checkbox so that it doesn't fall off the screenshot? Maybe since `Checkbox 6` already has long text that wraps, we could move that and the disabled one up?

```javascript
{
  type: 'checkbox',
  label: 'Checkbox 1',
  value: 'value1',
  checked: true,
},

{
  type: 'checkbox',
  label: 'Checkbox 2',
  value: 'value2',
  disabled: true,
},

{
  type: 'checkbox',
  label: 'Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3 Checkbox 3',
  value: 'value3',
},

{
  type: 'checkbox',
  label: 'Checkbox 4',
  value: 'value4',
},

{
  type: 'checkbox',
  label: 'Checkbox 5',
  value: 'value5',
},

{
  type: 'checkbox',
  label: 'Checkbox 6',
  value: 'value6',
},
```